components/VaultBox.tsx

```diff
@@ -261,6 +261,32 @@ function	VaultBox({vault, settings, noStrategies}: {vault: any, settings: TSetti
 				</section>
 			)}
 
+			{Object.keys(aggregatedData[toAddress(vault.address)]?.missingTranslations).length !== 0 && settings.shouldShowMissingTranslations ? (
```

Comment:
Crash on Optimism here, this should fix it:

```suggestion
			{Object.keys((aggregatedData?.[toAddress(vault.address)]?.missingTranslations) || []).length !== 0 && settings.shouldShowMissingTranslations ? (
```